# Comprimentos de cada parte do braço (cm)
length_OMB: 23.5
length_COT: 24.5
length_PUN: 6.5

# Correções angulares (graus)
correction_OMB: 0.0
correction_COT: 0.0
correction_PUN: 10.0

yaw_acceptable_error: 6.0 # Erro angular aceitável (graus)
Kp: 0.299 # Constante de proporção do PID do yaw do robô

debug: true # Mostrar ou não logs de debug